corva/models/stream.py

```diff
@@ -129,24 +130,22 @@ def asset_id(self) -> Optional[int]:
         return values
 
     @staticmethod
-    def from_raw_event(event: Union[str, List], **kwargs) -> List[StreamEvent]:
+    def from_raw_event(event: List[dict], **kwargs) -> List[StreamEvent]:
         app_key = kwargs['app_key']
 
-        parse = pydantic.parse_obj_as
-        if isinstance(event, str):
-            parse = pydantic.parse_raw_as
+        event = copy.deepcopy(
+            event
+        )  # deepcopy an event as app_key field will be added to each dict
 
-        event_dicts = parse(List[dict], event)  # type: List[dict]
-
-        for event_dict in event_dicts:
+        for event_dict in event:
             if 'app_key' in event_dict:
                 raise ValueError(
-                    "app_key can't be set manually, it is extracted from env and set automatically."
+                    'app_key can\'t be set manually, it is extracted from env and set automatically.'
```

Comment:
OK my dude, we don't have to be allergic to double quotes 😄

Reply:
it changed smh:) anyways after adding black to ci everything will be double quotes as it is a default